resource-managers/kubernetes/docker-minimal-bundle/src/main/docker/shuffle/Dockerfile

```diff
@@ -0,0 +1,39 @@
+#
```

Comment:
Nit: `src/main/docker/shuffle` -> `src/main/docker/shuffle-service` for a tad more clarity.